spacer/config.py

```diff
@@ -116,6 +116,7 @@ def get_local_model_path():
 
 # Check access to select which tests to run.
 HAS_CAFFE = importlib.util.find_spec("caffe") is not None
+HAS_TORCH = importlib.util.find_spec("torch") is not None
```

Comment:
We actually don't need this. Torch is available in all builds. It's just caffe that is finicky since only available in Docker.

Reply:
Good to know this, Pytorch availability checking is removed from `config.py`